pandas/tests/tools/test_to_datetime.py

```diff
@@ -177,6 +177,28 @@ def test_to_datetime_format_YYYYMMDD_overflow(self, input_s, expected):
         result = to_datetime(input_s, format="%Y%m%d", errors="coerce")
         tm.assert_series_equal(result, expected)
 
+    @pytest.mark.parametrize(
+        "data, format, expected",
+        [
+            ([pd.NA], "%Y%m%d%H%M%S", DatetimeIndex(["NaT"])),
+            ([pd.NA], None, DatetimeIndex(["NaT"])),
+            (
+                [pd.NA, "20210202202020"],
+                "%Y%m%d%H%M%S",
+                DatetimeIndex(["NaT", "2021-02-02 20:20:20"]),
+            ),
```

Comment:
this does't handle the errors= argument differences (well it likely handles 'coerce' and 'errors', but not 'ignore'

Reply:
Hi @jreback - added another test to specifically test all `errors` arguments

Reply:
> added another test to specifically test all errors arguments

im not seeing this, where is it?

Reply:
I had reverted back to the cython solution. Let me know if I should add a test for the error args.